docs/guides/wifi-bluetooth.md

```diff
@@ -5,7 +5,7 @@
 This page is a step by step guide to get Wi-Fi and Bluetooth working on T2 Macs. This guide is also applicable to **iMac19,1** and **iMac19,2**, which are T1 Intel Macs. This guide is NOT applicable for the rest of the T1 and older Intel Macs.
 
 !!! Bug "Unable to connect to Wi-Fi even after entering correct password"
-    Due to a [regression](https://lists.infradead.org/pipermail/hostap/2024-August/042893.html) since **wpa_supplicant 2.11**, Wi-Fi on broadcom chips is broken, thus affecting the T2 Macs as well. Please use [iwd](https://wiki.archlinux.org/title/NetworkManager#Using_iwd_as_the_Wi-Fi_backend) or [disable offloading](https://bbs.archlinux.org/viewtopic.php?pid=2189161#p2189161) until the issue is resolved. Currently, only Arch Linux and EndeavourOS are affected.
+    Due to a [regression](https://lists.infradead.org/pipermail/hostap/2024-August/042893.html) since **wpa_supplicant 2.11**, Wi-Fi on broadcom chips is broken, thus affecting the T2 Macs as well. Please use [iwd](https://wiki.archlinux.org/title/NetworkManager#Using_iwd_as_the_Wi-Fi_backend) or [disable offloading](https://bbs.archlinux.org/viewtopic.php?pid=2189161#p2189161) by adding `brcmfmac.feature_disable=0x82000` to your kernel parameters until the issue is resolved. Currently, only Arch Linux and EndeavourOS are affected.
```

Comment:
Just remove the link. This link is more confusing that useful. Instead make **disable offloading** bold.

Reply:
Also make it 

"please use iwd as a backend or disable offloading...."

Reply:
I suggest making suggestions using the provided mechanism in GitHub to avoid any mistakes.